workspaces/src/network/sandbox.rs

```diff
@@ -142,11 +146,15 @@ impl TopLevelAccountCreator for Sandbox {
     async fn create_tla_and_deploy(
         &self,
         worker: Worker<dyn Network>,
-        id: AccountId,
+        account_prefix: String,
```

Comment:
TLA stands for "top-level account", so this does not make sense.

The TLA account creation transaction must be signed as registrar account. Could you, please, generate a sandbox home folder and attach the genesis.json file here? It should [have `registrar_account_id`](https://github.com/near/nearcore/blob/677a59fdcc11fcd179ff4719ded4c41226e347f1/utils/mainnet-res/res/mainnet_genesis.json#L207-L210), and have it in the `"records"` together with the access key, so I hope to see if we control that account and if not, we should implement it, it we already do, just use that instead of the `root_signer`.

Reply:
ok, I will try it

Reply:
I registered `registerar` account's key and initial balance in `genesis.json` and create `registerar.json` file as registerar's wallet. but two test failed, I will check